build.gradle

```diff
@@ -56,6 +61,26 @@ if (project.hasProperty("maxParallelForks")) {
   ext.maxParallelForks = 3
 }
 
+animalsniffer {
+  sourceSets = [sourceSets.main]
+}
+
+findbugs {
+  // Some errors are logged due to https://github.com/gradle/gradle/issues/1094, but Findbugs checks are working fine
+  toolVersion '3.0.0'
+  sourceSets = [sourceSets.main]
+}
+
+codenarc {
+  // Some errors are logged due to https://github.com/gradle/gradle/issues/1094, but Findbugs checks are working fine
+  toolVersion '0.27.0'
+  sourceSets = [sourceSets.test]
+}
+
+configurations {
+  // https://github.com/gradle/gradle/issues/1305
+  compile.exclude group: 'com.ibm.icu', module: 'icu4j'
```

Comment:
Are you sure I don't need this dependency for running Jenkins? I am also somewhat afraid the generated HPI file could by wrong by doing this. What does the Maven build do to support animal sniffer?